src/site/content/en/blog/gamepad/index.md

```diff
@@ -275,38 +274,25 @@ or moving any of its axis.
 
 {% Glitch { id: 'gamepad-demo', path: 'script.js', height: 1000, allow: 'gamepad' } %}
 
-## Bonus: play Chrome dino on a Nintendo Switch
-
-The Nintendo Switch contains a
-[hidden browser](https://switchbrew.org/wiki/Internet_Browser#WifiWebAuthApplet),
-which serves for logging in to Wi-Fi networks behind a captive portal.
-The browser is pretty barebones and does not have a URL bar,
-but, once you have navigated to a page, it is fully usable.
-When doing a connection test in system settings, the Switch will detect that the captive portal
-is present and display an error for it when the response for
-[http://conntest.nintendowifi.net/](http://conntest.nintendowifi.net/)
-does not include the `X-Organization: Nintendo` HTTP header.
-I can make creative use of this by pointing the Switch to a DNS server
-that simulates a captive portal that then redirects to a search engine.
-
-1. Go to **System Settings** and then **Internet Settings** and
-   find the Wi-Fi network that your Switch is connected to. Tap **Change Settings**.
-1. Find the section with the **DNS Settings** and add
-   [45.55.142.122](http://45.55.142.122) as a new **Primary DNS**.
-   Note that this DNS server is *not operated by Google*
-   but a [third-party](https://www.switchbru.com/dns/), so proceed at your own risk.
-1. **Save** the settings and then tap **Connect to This Network**.
-1. The Switch will tell you that **Registration is required to use this network**. Tap **Next**.
-1. On the page that opens, make your way to **Google**.
-1. Search for **"chrome dino tomayac"**. This should lead you to
-   [https://github.com/tomayac/chrome-dino-gamepad](https://github.com/tomayac/chrome-dino-gamepad).
-1. On the right-hand side in the **About** section, find the link to
-   [https://tomayac.github.io/chrome-dino-gamepad/](https://tomayac.github.io/chrome-dino-gamepad/).
-   Enjoy!
-1. 🚨 For regular Switch online services to work again,
-   turn your DNS settings back to **Automatic**.
-   Conveniently, the Switch remembers previous manual DNS settings,
-   so you can easily toggle between **Automatic** and **Manual**.
+## Bonus: play Chrome dino on web.dev
+
+You can play
+[Chrome dino](https://tomayac.github.io/chrome-dino-gamepad/)
+with your gamepad on this very site.
+The source code is available
+[on GitHub](https://github.com/tomayac/chrome-dino-gamepad).
+Check out the gamepad polling implementation
+[in `trex-runner.js`](https://github.com/tomayac/chrome-dino-gamepad/blob/885eb6134805345bf31eeb9971830adeb84747ab/trex-runner.js#L529-L571)
+and note how it is emulating key presses.
+
+<div style="height: 420px; width: 100%;">
```

Comment:
Nit: Use "Iframe" web.dev component

Reply:
I don't think there exists one: https://web.dev/handbook/web-dev-components/

Reply:
There's one https://github.com/GoogleChrome/web.dev/blob/master/src/site/_includes/components/IFrame.js used by glitch https://github.com/GoogleChrome/web.dev/blob/master/src/site/_includes/components/Glitch.js

Reply:
With the still required div wrapper for responsiveness it's honestly not adding much value. Maybe that's why it's not documented. It makes a lot of sense for Glitch embeds, but not bare iframes.

Reply:
With "{Iframe}", `loading="lazy"` is automatically added ;) Your choice!

Reply:
Just done that manually in https://github.com/GoogleChrome/web.dev/pull/4181/commits/814ecbcfac704ca0c116e8364da5e6cb90f3a117.